src/pseudopeople/noise_entities.py

```diff
@@ -20,7 +20,16 @@ class __NoiseTypes(NamedTuple):
         "do_not_respond", noise_functions.apply_do_not_respond
     )
     omit_row: RowNoiseType = RowNoiseType("omit_row", noise_functions.omit_rows)
-    # duplicate_row: RowNoiseType = RowNoiseType("duplicate_row", noise_functions.duplicate_rows)
+    duplicate_with_guardian: RowNoiseType = RowNoiseType(
```

Comment:
Did you confirm this is to happen after do_not_respond and omit_row?

Reply:
Yes we have a test which tests the order of noise.